core/src/App.html

```diff
@@ -226,6 +240,20 @@
               console.error('goBack() not possible, no preserved views found.');
             }
           }
+
+          if ('luigi.displayAlert' === e.data.msg) {
+            if (e.data.detail) {
+              this.set({ alertMessage: e.data.detail });
+            } else {
+              console.error(
+                'Error alert has been triggered but no message was provided in "detail" property'
```

Comment:
I wouldn't show to the user an error saying that developers misconfigured something. That wouldn't tell the user anything, but could cause a confusion. In my opinion it would be sufficient that this information is visible in the console and in the UI there is a more generic message. 
We can discuss it with the team.